src/bereikbaarheid/apps.py

```diff
@@ -4,3 +4,7 @@
 class BereikbaarheidConfig(AppConfig):
     default_auto_field = "django.db.models.BigAutoField"
     name = "bereikbaarheid"
+
+    def ready(self):
+        # Implicitly connect signal handlers decorated with @receiver.
+        pass
```

Comment:
een `pass` heeft eigenlijk nooit iets te zoeken in productie code. het is een placeholder voor iets dat nog moet worden afgemaakt. wat is de bedoeling hier?

Reply:
dit komt rechtstreeks uit https://docs.djangoproject.com/en/4.2/topics/signals/ onder het kopje "Where should this code live?"

Reply:
autoflake actie -> aangepast met #NOQA